src/main.py

```diff
@@ -1,38 +1,32 @@
-# Based on https://randomnerdtutorials.com/esp32-esp8266-micropython-web-server/
-import esp
-import gc
-from machine import Pin
-import network
-import time
+# Based on https://randomnerdtutorials.com/esp32-esp8266-micropython-web
+# -server/
 try:
     import usocket as socket
+
     print("Using usocket")
-except:
+except ImportError:
     import socket
+
     print("Using socket")
-    
 
-def render_page():
+from blinker import Blinker
+
+
+def render_page(sensors):
     """
     This function is called to render the webserver page
     """
-    # we can define variable on outside scope in a conditional experession
```

Comment:
This logic is now handled in the `Blinker` class, which implements the LED sensor